From: KevinDuan <duankaiwen@kylinos.cn>
Date: Mon, 29 Jun 2026 09:51:26 +0000
Subject: =?utf-8?b?VHJhbnNsYXRlZCB1c2luZyBXZWJsYXRlICjmnJ3pspzor60gKGtvX0tQ?=
 =?utf-8?b?KSk=?=

Currently translated at 100.0% (370 of 370 strings)

Translation: openkylin-huanghe/ukui-screensaver
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/ukui-screensaver/ko_KP/
---
 i18n_ts/ko_KP.ts | 422 +++++++++++++++++++++++++++----------------------------
 1 file changed, 211 insertions(+), 211 deletions(-)

diff --git a/i18n_ts/ko_KP.ts b/i18n_ts/ko_KP.ts
index 6a8a1bb..14f5891 100644
--- a/i18n_ts/ko_KP.ts
+++ b/i18n_ts/ko_KP.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../src/widgets/agreementwindow.cpp" line="47"/>
         <source>I know</source>
-        <translation type="unfinished"></translation>
+        <translation>난 이미 알고 있어</translation>
     </message>
 </context>
 <context>
@@ -26,7 +26,7 @@
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="1662"/>
         <source>Retry</source>
-        <translation type="unfinished"></translation>
+        <translation>다시 시도해 보세요</translation>
     </message>
     <message>
         <source>UnLock</source>
@@ -51,7 +51,7 @@
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="996"/>
         <source>Password Prompt:%1</source>
-        <translation type="unfinished"></translation>
+        <translation>비밀번호 힌트: %1</translation>
     </message>
     <message>
         <source>Account locked %1 minutes due to %2 fail attempts</source>
@@ -60,17 +60,17 @@
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="1738"/>
         <source>Please try again in %1 minutes.</source>
-        <translation type="unfinished"></translation>
+        <translation>1분 후에 다시 시도해 주세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="1746"/>
         <source>Please try again in %1 seconds.</source>
-        <translation type="unfinished"></translation>
+        <translation>1초 후에 다시 시도해 주세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="1754"/>
         <source>Account locked permanently.</source>
-        <translation type="unfinished"></translation>
+        <translation>계정은 영구적으로 잠겼습니다</translation>
     </message>
     <message>
         <source>Verify face recognition or enter password to unlock</source>
@@ -79,7 +79,7 @@
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="500"/>
         <source>Guest</source>
-        <translation type="unfinished"></translation>
+        <translation>관광객</translation>
     </message>
     <message>
         <source>Press fingerprint or enter password to unlock</source>
@@ -100,33 +100,33 @@
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="498"/>
         <source>Enter username</source>
-        <translation type="unfinished"></translation>
+        <translation>로그인 사용자 이름을 입력하세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="791"/>
         <location filename="../src/widgets/authdialog.cpp" line="823"/>
         <source>Facial recognition to unlock</source>
-        <translation type="unfinished"></translation>
+        <translation>얼굴 인식 잠금 해제</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="794"/>
         <source>Fingerprint recognition to unlock</source>
-        <translation type="unfinished"></translation>
+        <translation>지문 인식 및 잠금 해제</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="797"/>
         <source>Voiceprint recognition to unlock</source>
-        <translation type="unfinished"></translation>
+        <translation>음성 인식 및 잠금 해제</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="800"/>
         <source>Finger vein recognition to unlock</source>
-        <translation type="unfinished"></translation>
+        <translation>손가락 정맥 잠금 해제 인식</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="803"/>
         <source>Iris recognition to unlock</source>
-        <translation type="unfinished"></translation>
+        <translation>홍채 잠금 해제 인식</translation>
     </message>
     <message>
         <source>Use the bound wechat scanning code to unlock</source>
@@ -135,33 +135,33 @@
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="806"/>
         <source>wechat</source>
-        <translation type="unfinished"></translation>
+        <translation>위챗</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="810"/>
         <source>Use the bound %1 scanning code to unlock</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 코드를 스캔해 잠금 해제</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="1125"/>
         <location filename="../src/widgets/authdialog.cpp" line="1147"/>
         <source>The authentication has been cancelled!</source>
-        <translation type="unfinished"></translation>
+        <translation>자격증 취소!</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="1370"/>
         <source>Password:</source>
-        <translation type="unfinished"></translation>
+        <translation>비밀번호:</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="1373"/>
         <source>Input Password</source>
-        <translation type="unfinished"></translation>
+        <translation>비밀번호를 입력하세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="1378"/>
         <source>Username</source>
-        <translation type="unfinished"></translation>
+        <translation>사용자 이름:</translation>
     </message>
     <message>
         <source>User name input error!</source>
@@ -170,11 +170,11 @@
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="1576"/>
         <source>login</source>
-        <translation type="unfinished"></translation>
+        <translation>로그인</translation>
     </message>
     <message>
         <source>Failed to verify %1, please try another way to unlock</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 검증에 실패했습니다. 다른 잠금 해제 방법을 시도해 주세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2006"/>
@@ -183,100 +183,100 @@
         <location filename="../src/widgets/authdialog.cpp" line="2418"/>
         <location filename="../src/widgets/authdialog.cpp" line="2609"/>
         <source>Unable to verify %1, please enter password to unlock</source>
-        <translation type="unfinished"></translation>
+        <translation>%1을 확인할 수 없으니, 잠금 해제를 위해 비밀번호를 입력해 주세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2451"/>
         <source>Abnormal network</source>
-        <translation type="unfinished"></translation>
+        <translation>네트워크 이상 현상</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2455"/>
         <source>Acquisition failure</source>
-        <translation type="unfinished"></translation>
+        <translation>실패</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2466"/>
         <source>Face recognition waiting time out, please click refresh or enter the password to unlock.</source>
-        <translation type="unfinished"></translation>
+        <translation>얼굴 인식은 타임아웃을 기다립니다; 잠금 해제하려면 새로고침을 클릭하거나 비밀번호를 입력해 주세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2738"/>
         <source>FingerPrint</source>
-        <translation type="unfinished"></translation>
+        <translation>지문</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2740"/>
         <source>FingerVein</source>
-        <translation type="unfinished"></translation>
+        <translation>손가락 정맥</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2742"/>
         <source>Iris</source>
-        <translation type="unfinished"></translation>
+        <translation>아이리스</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2744"/>
         <source>Face</source>
-        <translation type="unfinished"></translation>
+        <translation>얼굴 인식</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2746"/>
         <source>VoicePrint</source>
-        <translation type="unfinished"></translation>
+        <translation>음성 지문</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2748"/>
         <location filename="../src/widgets/authdialog.cpp" line="2750"/>
         <location filename="../src/widgets/authdialog.cpp" line="2772"/>
         <source>Ukey</source>
-        <translation type="unfinished"></translation>
+        <translation>보안 키</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2752"/>
         <location filename="../src/widgets/authdialog.cpp" line="2774"/>
         <source>QRCode</source>
-        <translation type="unfinished"></translation>
+        <translation>QR 코드</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2762"/>
         <source>fingerprint</source>
-        <translation type="unfinished"></translation>
+        <translation>지문</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2764"/>
         <source>fingervein</source>
-        <translation type="unfinished"></translation>
+        <translation>손가락 정맥</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2766"/>
         <source>iris</source>
-        <translation type="unfinished"></translation>
+        <translation>아이리스</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2768"/>
         <source>face</source>
-        <translation type="unfinished"></translation>
+        <translation>얼굴</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="2770"/>
         <source>voiceprint</source>
-        <translation type="unfinished"></translation>
+        <translation>음성 지문</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="3067"/>
         <source>Other</source>
-        <translation type="unfinished"></translation>
+        <translation>기타</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="3003"/>
         <source>Two-Factor OTP Dynamic Password Authentication</source>
-        <translation type="unfinished"></translation>
+        <translation>이중 인증 OTP 동적 비밀번호 인증</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="3006"/>
         <source>Two-Factor Security Key Authentication</source>
-        <translation type="unfinished"></translation>
+        <translation>이중 보안 키 인증</translation>
     </message>
     <message>
         <source>Use the bound wechat scanning code or enter the password to log in</source>
@@ -285,7 +285,7 @@
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="950"/>
         <source>Password cannot be empty</source>
-        <translation type="unfinished"></translation>
+        <translation>비밀번호는 비어 있을 수 없습니다</translation>
     </message>
     <message>
         <source>Failed to verify %1, please enter password.</source>
@@ -299,7 +299,7 @@
         <location filename="../src/widgets/authdialog.cpp" line="2435"/>
         <location filename="../src/widgets/authdialog.cpp" line="2439"/>
         <source>Failed to verify %1, you still have %2 verification opportunities</source>
-        <translation type="unfinished"></translation>
+        <translation>만약 1%에서 검증에 실패해도 시도할 기회가 두 번 남아 있습니다</translation>
     </message>
     <message>
         <source>Biometric/code scan authentication failed too many times, please enter the password.</source>
@@ -330,7 +330,7 @@
         <location filename="../src/widgets/authdialog.cpp" line="1488"/>
         <location filename="../src/widgets/authdialog.cpp" line="1492"/>
         <source>Authentication failure, Please try again</source>
-        <translation type="unfinished"></translation>
+        <translation>인증 실패, 다시 시도해 주세요</translation>
     </message>
     <message>
         <source>Use the bound wechat scanning code or enter the password to unlock</source>
@@ -339,13 +339,13 @@
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="424"/>
         <source>Enter the ukey password</source>
-        <translation type="unfinished"></translation>
+        <translation>보안 키와 비밀번호를 입력하세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="439"/>
         <location filename="../src/widgets/authdialog.cpp" line="2966"/>
         <source>Insert the ukey into the USB port</source>
-        <translation type="unfinished"></translation>
+        <translation>보안 키를 USB 포트에 꽂아 주세요</translation>
     </message>
     <message>
         <source>Password </source>
@@ -354,7 +354,7 @@
     <message>
         <location filename="../src/widgets/authdialog.cpp" line="1653"/>
         <source>Login</source>
-        <translation type="unfinished"></translation>
+        <translation>로그인</translation>
     </message>
     <message>
         <source>Biometric Authentication</source>
@@ -384,14 +384,14 @@
         <location filename="../src/widgets/batterywidget.cpp" line="255"/>
         <location filename="../src/widgets/batterywidget.cpp" line="263"/>
         <source>Charging</source>
-        <translation type="unfinished"></translation>
+        <translation>충전</translation>
     </message>
     <message>
         <location filename="../src/widgets/batterywidget.cpp" line="222"/>
         <location filename="../src/widgets/batterywidget.cpp" line="292"/>
         <location filename="../src/widgets/batterywidget.cpp" line="299"/>
         <source>Discharging</source>
-        <translation type="unfinished"></translation>
+        <translation>배출</translation>
     </message>
     <message>
         <location filename="../src/widgets/batterywidget.cpp" line="186"/>
@@ -400,23 +400,23 @@
         <location filename="../src/widgets/batterywidget.cpp" line="245"/>
         <location filename="../src/widgets/batterywidget.cpp" line="253"/>
         <source>Fully Charged</source>
-        <translation type="unfinished"></translation>
+        <translation>이미 꽉 찼어</translation>
     </message>
     <message>
         <location filename="../src/widgets/batterywidget.cpp" line="130"/>
         <source>PowerMode</source>
-        <translation type="unfinished"></translation>
+        <translation>전원 모드</translation>
     </message>
     <message>
         <location filename="../src/widgets/batterywidget.cpp" line="132"/>
         <source>BatteryMode</source>
-        <translation type="unfinished"></translation>
+        <translation>배터리 모드</translation>
     </message>
     <message>
         <location filename="../src/widgets/batterywidget.cpp" line="173"/>
         <location filename="../src/widgets/batterywidget.cpp" line="231"/>
         <source>No battery detected</source>
-        <translation type="unfinished"></translation>
+        <translation>배터리 감지되지 않음</translation>
     </message>
     <message>
         <location filename="../src/widgets/batterywidget.cpp" line="188"/>
@@ -424,7 +424,7 @@
         <location filename="../src/widgets/batterywidget.cpp" line="208"/>
         <location filename="../src/widgets/batterywidget.cpp" line="322"/>
         <source>Not Charging</source>
-        <translation type="unfinished"></translation>
+        <translation>요금 청구 안 했어요</translation>
     </message>
     <message>
         <location filename="../src/widgets/batterywidget.cpp" line="261"/>
@@ -516,38 +516,38 @@ remaining</source>
     <message>
         <location filename="../src/widgets/binddialog.cpp" line="91"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>닫습니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/binddialog.cpp" line="103"/>
         <location filename="../src/widgets/binddialog.cpp" line="302"/>
         <source>Ok</source>
-        <translation type="unfinished"></translation>
+        <translation>확인해</translation>
     </message>
     <message>
         <location filename="../src/widgets/binddialog.cpp" line="107"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>취소됨</translation>
     </message>
     <message>
         <location filename="../src/widgets/binddialog.cpp" line="119"/>
         <source>Mfa authentication is on, please bind OTP code</source>
-        <translation type="unfinished"></translation>
+        <translation>시스템이 2단계 인증을 활성화했습니다. 2단계 인증 비밀번호를 묶어 주세요.</translation>
     </message>
     <message>
         <location filename="../src/widgets/binddialog.cpp" line="125"/>
         <source>Go to Bind</source>
-        <translation type="unfinished"></translation>
+        <translation>가서 제본해</translation>
     </message>
     <message>
         <location filename="../src/widgets/binddialog.cpp" line="174"/>
         <source>Bind</source>
-        <translation type="unfinished"></translation>
+        <translation>제본법</translation>
     </message>
     <message>
         <location filename="../src/widgets/binddialog.cpp" line="307"/>
         <source>OTP added successfully</source>
-        <translation type="unfinished"></translation>
+        <translation>동적 비밀번호 성공적으로 추가됨</translation>
     </message>
 </context>
 <context>
@@ -628,7 +628,7 @@ remaining</source>
         <location filename="../src/widgets/blockwidget.cpp" line="152"/>
         <location filename="../src/widgets/blockwidget.cpp" line="154"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>취소됨</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="69"/>
@@ -638,7 +638,7 @@ remaining</source>
         <location filename="../src/widgets/blockwidget.cpp" line="188"/>
         <location filename="../src/widgets/blockwidget.cpp" line="189"/>
         <source>Confirm</source>
-        <translation type="unfinished"></translation>
+        <translation>확인해</translation>
     </message>
     <message>
         <source>still reboot</source>
@@ -652,12 +652,12 @@ remaining</source>
         <location filename="../src/widgets/blockwidget.cpp" line="228"/>
         <location filename="../src/widgets/blockwidget.cpp" line="254"/>
         <source>If you do not perform any operation, the system will automatically %1 after %2 seconds.</source>
-        <translation type="unfinished"></translation>
+        <translation>아무것도 하지 않으면 시스템이 2초 후에 자동으로 1% 성능을 발휘합니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="366"/>
         <source>Please wait while applications close and the computer reboot.</source>
-        <translation type="unfinished"></translation>
+        <translation>앱 종료 및 재시작</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="367"/>
@@ -665,22 +665,22 @@ remaining</source>
         <location filename="../src/widgets/blockwidget.cpp" line="377"/>
         <location filename="../src/widgets/blockwidget.cpp" line="386"/>
         <source>Cancel to complete your work and avoid data loss.</source>
-        <translation type="unfinished"></translation>
+        <translation>데이터 손실을 피하려면 &quot;취소&quot;를 클릭하여 반환한 후 원하는 작업을 완료할 수 있습니다.</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="371"/>
         <source>Please wait while applications close and the computer shutdown.</source>
-        <translation type="unfinished"></translation>
+        <translation>앱을 종료하고 기기를 종료하기</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="373"/>
         <source>shutdown</source>
-        <translation type="unfinished"></translation>
+        <translation>폐쇄</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="376"/>
         <source>Please wait while applications close and the computer suspend.</source>
-        <translation type="unfinished"></translation>
+        <translation>앱 종료 후 잠들기</translation>
     </message>
     <message>
         <source>still suspend</source>
@@ -689,7 +689,7 @@ remaining</source>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="381"/>
         <source>Please wait while applications close and the computer hibernation.</source>
-        <translation type="unfinished"></translation>
+        <translation>앱을 닫고 최대 절전 모드에 들어가기</translation>
     </message>
     <message>
         <source>still hibernate</source>
@@ -698,7 +698,7 @@ remaining</source>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="385"/>
         <source>Please wait while applications close and the computer logging out.</source>
-        <translation type="unfinished"></translation>
+        <translation>앱 종료 및 로그아웃</translation>
     </message>
     <message>
         <source>still logout</source>
@@ -707,27 +707,27 @@ remaining</source>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="270"/>
         <source>The following programs prevent restarting, you can click &quot;Cancel&quot; and then close these programs.</source>
-        <translation type="unfinished"></translation>
+        <translation>다음 프로그램들은 재시작을 차단합니다; &quot;취소&quot;를 클릭한 후 종료할 수 있습니다.</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="274"/>
         <source>The following programs prevent the shutdown, you can click &quot;Cancel&quot; and then close these programs.</source>
-        <translation type="unfinished"></translation>
+        <translation>다음 프로그램들은 종료를 차단합니다; &quot;취소&quot;를 클릭한 후 이 프로그램들을 종료할 수 있습니다.</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="279"/>
         <source>The following programs prevent suspend, you can click &quot;Cancel&quot; and then close these programs.</source>
-        <translation type="unfinished"></translation>
+        <translation>다음 프로그램들은 절전 차단을 차단합니다; &quot;취소&quot;를 클릭한 후 이 프로그램들을 닫을 수 있습니다.</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="282"/>
         <source>The following programs prevent hibernation, you can click &quot;Cancel&quot; and then close these programs.</source>
-        <translation type="unfinished"></translation>
+        <translation>다음 프로그램들은 절전 절전 모드를 차단합니다; &quot;취소&quot;를 클릭한 후 이 프로그램들을 종료할 수 있습니다.</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="286"/>
         <source>The following programs prevent you from logging out, you can click &quot;Cancel&quot; and then close them.</source>
-        <translation type="unfinished"></translation>
+        <translation>다음 프로그램들은 로그아웃을 차단합니다; &quot;취소&quot;를 클릭한 후 이 프로그램들을 종료할 수 있습니다.</translation>
     </message>
     <message>
         <source>This app prevents your computer from shutting down.</source>
@@ -736,49 +736,49 @@ remaining</source>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="291"/>
         <source>The following programs prevent you restarting, you can click &quot;Cancel&quot; and then close them,                 or click &apos;OK&apos; to force close and proceed.</source>
-        <translation type="unfinished"></translation>
+        <translation>다음 프로그램들은 재시작을 차단합니다; &quot;취소&quot;를 클릭하면 이 프로그램들을 닫거나, &quot;확인&quot;을 클릭하여 강제 종료 후 계속 진행할 수 있습니다.</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="162"/>
         <location filename="../src/widgets/blockwidget.cpp" line="235"/>
         <source>Still reboot</source>
-        <translation type="unfinished"></translation>
+        <translation>여전히 재부팅 중입니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="166"/>
         <location filename="../src/widgets/blockwidget.cpp" line="237"/>
         <source>Still shutdown</source>
-        <translation type="unfinished"></translation>
+        <translation>여전히 전원이 꺼져 있습니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="169"/>
         <source>Still suspend</source>
-        <translation type="unfinished"></translation>
+        <translation>아직 자고 있어</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="172"/>
         <source>Still hibernate</source>
-        <translation type="unfinished"></translation>
+        <translation>아직 휴면 상태입니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="176"/>
         <source>Still logout</source>
-        <translation type="unfinished"></translation>
+        <translation>여전히 등록 취소 상태입니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="296"/>
         <source>The following programs prevent you shutdown, you can click &quot;Cancel&quot; and then close them,                 or click &apos;OK&apos; to force close and proceed.</source>
-        <translation type="unfinished"></translation>
+        <translation>다음 프로그램들은 종료를 방지합니다; &quot;취소&quot;를 클릭하면 이 프로그램들을 닫거나, &quot;확인&quot;을 눌러 강제 종료하고 계속할 수 있습니다.</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="301"/>
         <source>The following programs prevent you from logging out, you can click &quot;Cancel&quot; and then close them,                  or click &apos;OK&apos; to force close and proceed.</source>
-        <translation type="unfinished"></translation>
+        <translation>다음 프로그램들은 로그아웃을 차단합니다; &quot;취소&quot;를 클릭하면 이 프로그램들을 닫거나, &quot;확인&quot;을 클릭하면 강제 종료 후 계속할 수 있습니다.</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="368"/>
         <source>reboot</source>
-        <translation type="unfinished"></translation>
+        <translation>재시작</translation>
     </message>
     <message>
         <source>shutting down</source>
@@ -787,32 +787,32 @@ remaining</source>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="378"/>
         <source>sleep</source>
-        <translation type="unfinished"></translation>
+        <translation>잠 자세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="382"/>
         <source>hibernate</source>
-        <translation type="unfinished"></translation>
+        <translation>휴면</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="387"/>
         <source>log out</source>
-        <translation type="unfinished"></translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="410"/>
         <source>This app prevents your computer from %1.</source>
-        <translation type="unfinished"></translation>
+        <translation>이 앱은 당신을 차단합니다 %</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="465"/>
         <source>shut down</source>
-        <translation type="unfinished"></translation>
+        <translation>폐쇄</translation>
     </message>
     <message>
         <location filename="../src/widgets/blockwidget.cpp" line="468"/>
         <source>restart</source>
-        <translation type="unfinished"></translation>
+        <translation>재시작</translation>
     </message>
 </context>
 <context>
@@ -829,7 +829,7 @@ remaining</source>
         <location filename="../src/VirtualKeyboard/src/charswidget.cpp" line="115"/>
         <location filename="../src/VirtualKeyboard/src/charswidget.cpp" line="273"/>
         <source>More</source>
-        <translation type="unfinished"></translation>
+        <translation>더 보기</translation>
     </message>
     <message>
         <location filename="../src/VirtualKeyboard/src/charswidget.cpp" line="129"/>
@@ -942,7 +942,7 @@ remaining</source>
     <message>
         <location filename="../src/screensaver/customalbumviewer.cpp" line="20"/>
         <source>No photos detected, please re-select them</source>
-        <translation type="unfinished"></translation>
+        <translation>사진이 감지되지 않으니 다시 선택해 주세요</translation>
     </message>
 </context>
 <context>
@@ -2047,17 +2047,17 @@ remaining</source>
     <message>
         <location filename="../src/widgets/fullbackgroundwidget.cpp" line="415"/>
         <source>rebooting</source>
-        <translation type="unfinished"></translation>
+        <translation>재부팅</translation>
     </message>
     <message>
         <location filename="../src/widgets/fullbackgroundwidget.cpp" line="418"/>
         <source>shutdowning</source>
-        <translation type="unfinished"></translation>
+        <translation>폐쇄</translation>
     </message>
     <message>
         <location filename="../src/widgets/fullbackgroundwidget.cpp" line="421"/>
         <source>logouting</source>
-        <translation type="unfinished"></translation>
+        <translation>등록 취소</translation>
     </message>
 </context>
 <context>
@@ -2066,7 +2066,7 @@ remaining</source>
         <location filename="../src/widgets/iconedit.cpp" line="121"/>
         <location filename="../src/widgets/iconedit.cpp" line="238"/>
         <source>OK</source>
-        <translation type="unfinished"></translation>
+        <translation>복종</translation>
     </message>
 </context>
 <context>
@@ -2102,17 +2102,17 @@ remaining</source>
         <location filename="../src/VirtualKeyboard/src/kbtitle.cpp" line="46"/>
         <location filename="../src/VirtualKeyboard/src/kbtitle.cpp" line="83"/>
         <source>Suspended state</source>
-        <translation type="unfinished"></translation>
+        <translation>플로팅 상태</translation>
     </message>
     <message>
         <location filename="../src/VirtualKeyboard/src/kbtitle.cpp" line="57"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>닫습니다</translation>
     </message>
     <message>
         <location filename="../src/VirtualKeyboard/src/kbtitle.cpp" line="86"/>
         <source>Welt status</source>
-        <translation type="unfinished"></translation>
+        <translation>간선 인접 상태</translation>
     </message>
 </context>
 <context>
@@ -2335,17 +2335,17 @@ remaining</source>
     <message>
         <location filename="../src/lock-backend/lightdmhelper.cpp" line="141"/>
         <source>failed to start session.</source>
-        <translation type="unfinished"></translation>
+        <translation>시작 세션이 실패했습니다.</translation>
     </message>
     <message>
         <location filename="../src/lock-backend/lightdmhelper.cpp" line="235"/>
         <source>Login</source>
-        <translation type="unfinished"></translation>
+        <translation>로그인</translation>
     </message>
     <message>
         <location filename="../src/lock-backend/lightdmhelper.cpp" line="244"/>
         <source>Guest</source>
-        <translation type="unfinished"></translation>
+        <translation>관광객</translation>
     </message>
 </context>
 <context>
@@ -2366,13 +2366,13 @@ remaining</source>
         <location filename="../src/widgets/lockwidget.cpp" line="2346"/>
         <location filename="../src/widgets/lockwidget.cpp" line="2650"/>
         <source>SwitchUser</source>
-        <translation type="unfinished"></translation>
+        <translation>스위치 사용자</translation>
     </message>
     <message>
         <location filename="../src/widgets/lockwidget.cpp" line="2335"/>
         <location filename="../src/widgets/lockwidget.cpp" line="2610"/>
         <source>Power</source>
-        <translation type="unfinished"></translation>
+        <translation>전원 공급</translation>
     </message>
     <message>
         <source>Multiple user sessions are active. Restart may lead to unsaved data loss for other logged-in users.</source>
@@ -2386,7 +2386,7 @@ remaining</source>
         <location filename="../src/widgets/lockwidget.cpp" line="2338"/>
         <location filename="../src/widgets/lockwidget.cpp" line="2628"/>
         <source>VirtualKeyboard</source>
-        <translation type="unfinished"></translation>
+        <translation>가상 키보드</translation>
     </message>
     <message>
         <source>Multiple users are logged in at the same time.Are you sure you want to reboot this system?</source>
@@ -2404,41 +2404,41 @@ remaining</source>
         <location filename="../src/widgets/lockwidget.cpp" line="2352"/>
         <location filename="../src/widgets/lockwidget.cpp" line="2729"/>
         <source>SwitchSession</source>
-        <translation type="unfinished"></translation>
+        <translation>전환 세션</translation>
     </message>
     <message>
         <location filename="../src/widgets/lockwidget.cpp" line="2349"/>
         <source>Power Information</source>
-        <translation type="unfinished"></translation>
+        <translation>전력 정보</translation>
     </message>
     <message>
         <location filename="../src/widgets/lockwidget.cpp" line="1846"/>
         <source>Restart may lead to unsaved data loss for other logged-in users.</source>
-        <translation type="unfinished"></translation>
+        <translation>재시작하면 이 컴퓨터에 로그인한 사용자가 아직 저장되지 않은 데이터를 잃을 수 있습니다.</translation>
     </message>
     <message>
         <location filename="../src/widgets/lockwidget.cpp" line="1851"/>
         <source>Shut Down may lead to unsaved data loss for other logged-in users.</source>
-        <translation type="unfinished"></translation>
+        <translation>종료 시 이 컴퓨터에 로그인한 사용자가 아직 저장되지 않은 데이터를 잃을 수 있습니다.</translation>
     </message>
     <message>
         <location filename="../src/widgets/lockwidget.cpp" line="2342"/>
         <location filename="../src/widgets/lockwidget.cpp" line="2640"/>
         <source>Network</source>
-        <translation type="unfinished"></translation>
+        <translation>인터넷</translation>
     </message>
     <message>
         <location filename="../src/widgets/lockwidget.cpp" line="749"/>
         <location filename="../src/widgets/lockwidget.cpp" line="2355"/>
         <location filename="../src/widgets/lockwidget.cpp" line="2560"/>
         <source>system-monitor</source>
-        <translation type="unfinished"></translation>
+        <translation>시스템 모니터</translation>
     </message>
     <message>
         <location filename="../src/widgets/lockwidget.cpp" line="1305"/>
         <location filename="../src/widgets/lockwidget.cpp" line="2358"/>
         <source>Reset Pwd</source>
-        <translation type="unfinished"></translation>
+        <translation>비밀번호 재설정</translation>
     </message>
     <message>
         <source>%1 may cause users who have logged in to this computer to lose content that has not yet been stored,To still perform please click &quot;Confirm&quot;.</source>
@@ -2448,7 +2448,7 @@ remaining</source>
         <location filename="../src/widgets/lockwidget.cpp" line="1792"/>
         <location filename="../src/widgets/lockwidget.cpp" line="1847"/>
         <source>Restart</source>
-        <translation type="unfinished"></translation>
+        <translation>재시작</translation>
     </message>
     <message>
         <source>Multiple users are logged in at the same time.Are you sure you want to %1 this system?</source>
@@ -2466,7 +2466,7 @@ remaining</source>
         <location filename="../src/widgets/lockwidget.cpp" line="1797"/>
         <location filename="../src/widgets/lockwidget.cpp" line="1852"/>
         <source>Shut Down</source>
-        <translation type="unfinished"></translation>
+        <translation>폐쇄</translation>
     </message>
     <message>
         <source>Multiple user sessions are active. %1 may lead to unsaved data loss for other logged-in users.</source>
@@ -2476,27 +2476,27 @@ remaining</source>
         <location filename="../src/widgets/lockwidget.cpp" line="1791"/>
         <location filename="../src/widgets/lockwidget.cpp" line="1796"/>
         <source>%1 may lead to unsaved data loss for other logged-in users.</source>
-        <translation type="unfinished"></translation>
+        <translation>1%는 이 컴퓨터에 로그인한 사용자가 저장되지 않은 데이터를 잃게 할 수 있습니다.</translation>
     </message>
     <message>
         <location filename="../src/widgets/lockwidget.cpp" line="2799"/>
         <source>No battery detected</source>
-        <translation type="unfinished"></translation>
+        <translation>배터리 감지되지 않음</translation>
     </message>
     <message>
         <location filename="../src/widgets/lockwidget.cpp" line="2806"/>
         <source>Fully Charged (100%)</source>
-        <translation type="unfinished"></translation>
+        <translation>완전 채워짐 (100%)</translation>
     </message>
     <message>
         <location filename="../src/widgets/lockwidget.cpp" line="2808"/>
         <source>%1% available (plugged in)</source>
-        <translation type="unfinished"></translation>
+        <translation>%1% 사용 가능 (전원 연결 상태)</translation>
     </message>
     <message>
         <location filename="../src/widgets/lockwidget.cpp" line="2811"/>
         <source>%1% remaining</source>
-        <translation type="unfinished"></translation>
+        <translation>% 1% 이용 가능</translation>
     </message>
 </context>
 <context>
@@ -2515,42 +2515,42 @@ remaining</source>
     <message>
         <location filename="../src/widgets/loginoptionswidget.cpp" line="307"/>
         <source>Other</source>
-        <translation type="unfinished"></translation>
+        <translation>기타</translation>
     </message>
     <message>
         <location filename="../src/widgets/loginoptionswidget.cpp" line="1141"/>
         <source>FingerPrint</source>
-        <translation type="unfinished"></translation>
+        <translation>지문</translation>
     </message>
     <message>
         <location filename="../src/widgets/loginoptionswidget.cpp" line="1143"/>
         <source>FingerVein</source>
-        <translation type="unfinished"></translation>
+        <translation>손가락 정맥</translation>
     </message>
     <message>
         <location filename="../src/widgets/loginoptionswidget.cpp" line="1145"/>
         <source>Iris</source>
-        <translation type="unfinished"></translation>
+        <translation>아이리스</translation>
     </message>
     <message>
         <location filename="../src/widgets/loginoptionswidget.cpp" line="1147"/>
         <source>Face</source>
-        <translation type="unfinished"></translation>
+        <translation>얼굴 인식</translation>
     </message>
     <message>
         <location filename="../src/widgets/loginoptionswidget.cpp" line="1149"/>
         <source>VoicePrint</source>
-        <translation type="unfinished"></translation>
+        <translation>음성 지문</translation>
     </message>
     <message>
         <location filename="../src/widgets/loginoptionswidget.cpp" line="1151"/>
         <source>Ukey</source>
-        <translation type="unfinished"></translation>
+        <translation>보안 키</translation>
     </message>
     <message>
         <location filename="../src/widgets/loginoptionswidget.cpp" line="1153"/>
         <source>QRCode</source>
-        <translation type="unfinished"></translation>
+        <translation>QR 코드</translation>
     </message>
     <message>
         <source>Wechat</source>
@@ -2559,7 +2559,7 @@ remaining</source>
     <message>
         <location filename="../src/widgets/loginoptionswidget.cpp" line="730"/>
         <source>Identify device removed!</source>
-        <translation type="unfinished"></translation>
+        <translation>보정 장치가 제거되었습니다!</translation>
     </message>
 </context>
 <context>
@@ -2568,13 +2568,13 @@ remaining</source>
         <location filename="../src/screensaver/mprissaverwidget.cpp" line="230"/>
         <location filename="../src/screensaver/mprissaverwidget.cpp" line="419"/>
         <source>No music detected</source>
-        <translation type="unfinished"></translation>
+        <translation>노래가 감지되지 않음</translation>
     </message>
     <message>
         <location filename="../src/screensaver/mprissaverwidget.cpp" line="444"/>
         <location filename="../src/screensaver/mprissaverwidget.cpp" line="453"/>
         <source>Unknown</source>
-        <translation type="unfinished"></translation>
+        <translation>미상</translation>
     </message>
 </context>
 <context>
@@ -2590,13 +2590,13 @@ remaining</source>
         <location filename="../src/screensaver/musicfileinformation.cpp" line="199"/>
         <location filename="../src/screensaver/musicfileinformation.cpp" line="235"/>
         <source>Unknown singer</source>
-        <translation type="unfinished"></translation>
+        <translation>알려지지 않은 음악</translation>
     </message>
     <message>
         <location filename="../src/screensaver/musicfileinformation.cpp" line="200"/>
         <location filename="../src/screensaver/musicfileinformation.cpp" line="238"/>
         <source>Unknown album</source>
-        <translation type="unfinished"></translation>
+        <translation>앨범 내용은 알려지지 않았습니다</translation>
     </message>
 </context>
 <context>
@@ -2611,12 +2611,12 @@ remaining</source>
     <message>
         <location filename="../src/widgets/mynetworkwidget.cpp" line="54"/>
         <source>LAN</source>
-        <translation type="unfinished"></translation>
+        <translation>유선 네트워크</translation>
     </message>
     <message>
         <location filename="../src/widgets/mynetworkwidget.cpp" line="56"/>
         <source>WLAN</source>
-        <translation type="unfinished"></translation>
+        <translation>무선 근거리 지역 네트워크</translation>
     </message>
 </context>
 <context>
@@ -2630,7 +2630,7 @@ remaining</source>
         <location filename="../src/VirtualKeyboard/src/numberswidget.cpp" line="175"/>
         <location filename="../src/VirtualKeyboard/src/numberswidget.cpp" line="310"/>
         <source>Return</source>
-        <translation type="unfinished"></translation>
+        <translation>귀환</translation>
     </message>
 </context>
 <context>
@@ -2748,22 +2748,22 @@ remaining</source>
     <message>
         <location filename="../src/widgets/passwdcheckutil.cpp" line="159"/>
         <source>The password is shorter than %1 characters</source>
-        <translation type="unfinished"></translation>
+        <translation>비밀번호는 1자 미만이어야 합니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/passwdcheckutil.cpp" line="162"/>
         <source>The password contains less than %1 character classes</source>
-        <translation type="unfinished"></translation>
+        <translation>비밀번호는 1자 미만으로 구성되어 있습니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/passwdcheckutil.cpp" line="165"/>
         <source>The password is the same as the old one</source>
-        <translation type="unfinished"></translation>
+        <translation>옛 코드와 새 코드는 동일합니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/passwdcheckutil.cpp" line="168"/>
         <source>The password contains the user name in some form</source>
-        <translation type="unfinished"></translation>
+        <translation>비밀번호에는 사용자 이름이 포함되어 있습니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/passwdcheckutil.cpp" line="171"/>
@@ -2773,7 +2773,7 @@ remaining</source>
     <message>
         <location filename="../src/widgets/passwdcheckutil.cpp" line="174"/>
         <source>The password is too similar to the old one</source>
-        <translation type="unfinished"></translation>
+        <translation>새 암호는 이전 암호와 너무 비슷합니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/passwdcheckutil.cpp" line="177"/>
@@ -2840,19 +2840,19 @@ remaining</source>
         <location filename="../src/widgets/poweritemwidget.cpp" line="127"/>
         <location filename="../src/widgets/poweritemwidget.cpp" line="131"/>
         <source>Estimated </source>
-        <translation type="unfinished"></translation>
+        <translation>예상 </translation>
     </message>
     <message>
         <location filename="../src/widgets/poweritemwidget.cpp" line="124"/>
         <location filename="../src/widgets/poweritemwidget.cpp" line="127"/>
         <source> minutes</source>
-        <translation type="unfinished"></translation>
+        <translation> 몇 분</translation>
     </message>
     <message>
         <location filename="../src/widgets/poweritemwidget.cpp" line="127"/>
         <location filename="../src/widgets/poweritemwidget.cpp" line="131"/>
         <source> hours</source>
-        <translation type="unfinished"></translation>
+        <translation> 운영 시간</translation>
     </message>
 </context>
 <context>
@@ -2861,19 +2861,19 @@ remaining</source>
         <location filename="../src/widgets/powerlistwidget.cpp" line="342"/>
         <location filename="../src/widgets/powerlistwidget.h" line="148"/>
         <source>Hibernate</source>
-        <translation type="unfinished"></translation>
+        <translation>휴면</translation>
     </message>
     <message>
         <location filename="../src/widgets/powerlistwidget.cpp" line="343"/>
         <location filename="../src/widgets/powerlistwidget.h" line="149"/>
         <source>Turn off your computer, but the app stays open, When the computer is turned on, it can be restored to the state you left</source>
-        <translation type="unfinished"></translation>
+        <translation>컴퓨터를 끄면 앱은 계속 열려 있습니다. 켜면 남겨둔 상태를 복원할 수 있습니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/powerlistwidget.cpp" line="346"/>
         <location filename="../src/widgets/powerlistwidget.h" line="135"/>
         <source>Suspend</source>
-        <translation type="unfinished"></translation>
+        <translation>잠 자세요</translation>
     </message>
     <message>
         <source>The computer stays on, but consumes less power. The app stays open and can quickly wake up and revert to where you left off</source>
@@ -2883,24 +2883,24 @@ remaining</source>
         <location filename="../src/widgets/powerlistwidget.cpp" line="350"/>
         <location filename="../src/widgets/powerlistwidget.h" line="185"/>
         <source>Restart</source>
-        <translation type="unfinished"></translation>
+        <translation>재시작</translation>
     </message>
     <message>
         <location filename="../src/widgets/powerlistwidget.cpp" line="351"/>
         <source>Close all apps, and then restart your computer</source>
-        <translation type="unfinished"></translation>
+        <translation>모든 앱을 종료한 후 컴퓨터를 재시작하세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/powerlistwidget.cpp" line="353"/>
         <location filename="../src/widgets/powerlistwidget.h" line="161"/>
         <source>Shut Down</source>
-        <translation type="unfinished"></translation>
+        <translation>폐쇄</translation>
     </message>
     <message>
         <location filename="../src/widgets/powerlistwidget.cpp" line="358"/>
         <location filename="../src/widgets/powerlistwidget.h" line="124"/>
         <source>The current user logs out of the system, terminates the session, and returns to the login page</source>
-        <translation type="unfinished"></translation>
+        <translation>현재 사용자는 시스템에서 로그아웃되어 세션을 종료한 후 로그인 페이지로 돌아갑니다</translation>
     </message>
     <message>
         <source>Logout</source>
@@ -2914,48 +2914,48 @@ remaining</source>
         <location filename="../src/widgets/powerlistwidget.cpp" line="354"/>
         <location filename="../src/widgets/powerlistwidget.h" line="162"/>
         <source>Close all apps, and then shut down your computer</source>
-        <translation type="unfinished"></translation>
+        <translation>모든 앱을 종료한 후 컴퓨터를 종료하세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/powerlistwidget.cpp" line="360"/>
         <location filename="../src/widgets/powerlistwidget.h" line="110"/>
         <source>SwitchUser</source>
-        <translation type="unfinished"></translation>
+        <translation>스위치 사용자</translation>
     </message>
     <message>
         <location filename="../src/widgets/powerlistwidget.cpp" line="362"/>
         <location filename="../src/widgets/powerlistwidget.h" line="98"/>
         <source>LockScreen</source>
-        <translation type="unfinished"></translation>
+        <translation>잠금 화면</translation>
     </message>
     <message>
         <location filename="../src/widgets/powerlistwidget.cpp" line="356"/>
         <location filename="../src/widgets/powerlistwidget.h" line="122"/>
         <source>Log Out</source>
-        <translation type="unfinished"></translation>
+        <translation>취소</translation>
     </message>
     <message>
         <location filename="../src/widgets/powerlistwidget.cpp" line="347"/>
         <location filename="../src/widgets/powerlistwidget.h" line="136"/>
         <source>The computer stays on, but consumes less power, The app stays open and can quickly wake up and revert to where you left off</source>
-        <translation type="unfinished"></translation>
+        <translation>컴퓨터는 켜져 있지만 전력 소모가 줄어들고; 앱은 계속 열려 있어 빠르게 깨어나 떠난 상태를 복구할 수 있습니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/powerlistwidget.cpp" line="364"/>
         <location filename="../src/widgets/powerlistwidget.h" line="197"/>
         <source>UpgradeThenRestart</source>
-        <translation type="unfinished"></translation>
+        <translation>업데이트 및 재시작</translation>
     </message>
     <message>
         <location filename="../src/widgets/powerlistwidget.h" line="186"/>
         <source>Close all apps, turn off your computer, and then turn your computer back on</source>
-        <translation type="unfinished"></translation>
+        <translation>모든 앱을 닫고, 컴퓨터를 종료한 다음, 다시 켜세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/powerlistwidget.cpp" line="366"/>
         <location filename="../src/widgets/powerlistwidget.h" line="173"/>
         <source>UpgradeThenShutdown</source>
-        <translation type="unfinished"></translation>
+        <translation>업데이트 및 종료</translation>
     </message>
 </context>
 <context>
@@ -3038,47 +3038,47 @@ remaining</source>
     <message>
         <location filename="../src/lock-command/main.cpp" line="108"/>
         <source>The screensaver is active.</source>
-        <translation type="unfinished"></translation>
+        <translation>화면 보호기가 활성화되었습니다</translation>
     </message>
     <message>
         <location filename="../src/lock-command/main.cpp" line="110"/>
         <source>The screensaver is inactive.</source>
-        <translation type="unfinished"></translation>
+        <translation>화면 보호기가 활성화되지 않습니다</translation>
     </message>
     <message>
         <location filename="../src/common/biodefines.cpp" line="28"/>
         <source>FingerPrint</source>
-        <translation type="unfinished"></translation>
+        <translation>지문</translation>
     </message>
     <message>
         <location filename="../src/common/biodefines.cpp" line="30"/>
         <source>FingerVein</source>
-        <translation type="unfinished"></translation>
+        <translation>손가락 정맥</translation>
     </message>
     <message>
         <location filename="../src/common/biodefines.cpp" line="32"/>
         <source>Iris</source>
-        <translation type="unfinished"></translation>
+        <translation>아이리스</translation>
     </message>
     <message>
         <location filename="../src/common/biodefines.cpp" line="34"/>
         <source>Face</source>
-        <translation type="unfinished"></translation>
+        <translation>얼굴 인식</translation>
     </message>
     <message>
         <location filename="../src/common/biodefines.cpp" line="36"/>
         <source>VoicePrint</source>
-        <translation type="unfinished"></translation>
+        <translation>음성 지문</translation>
     </message>
     <message>
         <location filename="../src/common/biodefines.cpp" line="38"/>
         <source>Ukey</source>
-        <translation type="unfinished"></translation>
+        <translation>보안 키</translation>
     </message>
     <message>
         <location filename="../src/common/biodefines.cpp" line="40"/>
         <source>QRCode</source>
-        <translation type="unfinished"></translation>
+        <translation>QR 코드</translation>
     </message>
     <message>
         <location filename="../src/screensaver/musicdatabase.cpp" line="64"/>
@@ -3095,67 +3095,67 @@ remaining</source>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="80"/>
         <source>Security Question</source>
-        <translation type="unfinished"></translation>
+        <translation>보안 문제</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="81"/>
         <source>Kylin ID</source>
-        <translation type="unfinished"></translation>
+        <translation>기린 ID</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="160"/>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="359"/>
         <source>Enter new password</source>
-        <translation type="unfinished"></translation>
+        <translation>새 비밀번호 입력하세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="184"/>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="360"/>
         <source>Confirm your password</source>
-        <translation type="unfinished"></translation>
+        <translation>비밀번호를 확인해 주세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="187"/>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="322"/>
         <source>Inconsistency with pwd</source>
-        <translation type="unfinished"></translation>
+        <translation>새 비밀번호와 일관성 없어</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="215"/>
         <source>Confirm</source>
-        <translation type="unfinished"></translation>
+        <translation>확인해</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="225"/>
         <source>Cancel</source>
-        <translation type="unfinished"></translation>
+        <translation>취소됨</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="291"/>
         <source>Contains illegal characters!</source>
-        <translation type="unfinished"></translation>
+        <translation>불법 캐릭터가 등장합니다</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="385"/>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="456"/>
         <source>The service is abnormal</source>
-        <translation type="unfinished"></translation>
+        <translation>서비스 이상</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="393"/>
         <source>The SMS verification code has been sent to the mobile phone with the tail number %1 (Valid for 5 minutes)</source>
-        <translation type="unfinished"></translation>
+        <translation>SMS 인증 코드가 1%로 끝나는 번호로 전송되었습니다(유효 5분 이내)</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="401"/>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="406"/>
         <source>Failed to get the verification code</source>
-        <translation type="unfinished"></translation>
+        <translation>인증 코드를 얻지 못함</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="414"/>
         <source>The account does not exist, please re-enter it</source>
-        <translation type="unfinished"></translation>
+        <translation>계정은 존재하지 않으니 다시 입력해 주세요</translation>
     </message>
     <message>
         <source>The account does not exist, please complete the registration first</source>
@@ -3185,18 +3185,18 @@ remaining</source>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="529"/>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="557"/>
         <source>Verify Kylin ID Reset Password</source>
-        <translation type="unfinished"></translation>
+        <translation>키린 ID 재설정 비밀번호를 확인하세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="532"/>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="551"/>
         <source>Answer the security questions and reset your password</source>
-        <translation type="unfinished"></translation>
+        <translation>비밀번호 재설정을 위한 보안 질문에 답하세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/resetpwdwidget.cpp" line="586"/>
         <source>One or more wrong answers</source>
-        <translation type="unfinished"></translation>
+        <translation>하나 이상의 오답</translation>
     </message>
 </context>
 <context>
@@ -3397,7 +3397,7 @@ remaining</source>
         <location filename="../src/screensaver/screensaver.cpp" line="570"/>
         <location filename="../src/screensaver/screensaver.cpp" line="595"/>
         <source>Picture does not exist</source>
-        <translation type="unfinished"></translation>
+        <translation>그 사진은 존재하지 않습니다</translation>
     </message>
     <message>
         <location filename="../src/screensaver/screensaver.cpp" line="1567"/>
@@ -3428,33 +3428,33 @@ remaining</source>
         <location filename="../src/screensaver/screensaver.cpp" line="2292"/>
         <location filename="../src/screensaver/screensaver.cpp" line="2294"/>
         <source>Not connected to a network</source>
-        <translation type="unfinished"></translation>
+        <translation>네트워크 연결 안 됨</translation>
     </message>
     <message>
         <location filename="../src/screensaver/screensaver.cpp" line="2298"/>
         <location filename="../src/screensaver/screensaver.cpp" line="2300"/>
         <source>The weather update failed</source>
-        <translation type="unfinished"></translation>
+        <translation>날씨 업데이트 실패</translation>
     </message>
     <message>
         <location filename="../src/screensaver/screensaver.cpp" line="2386"/>
         <source>Smart Saver</source>
-        <translation type="unfinished"></translation>
+        <translation>스마트 화면 보호기</translation>
     </message>
     <message>
         <location filename="../src/screensaver/screensaver.cpp" line="2462"/>
         <source>Weather</source>
-        <translation type="unfinished"></translation>
+        <translation>날씨</translation>
     </message>
     <message>
         <location filename="../src/screensaver/screensaver.cpp" line="2490"/>
         <source>Music</source>
-        <translation type="unfinished"></translation>
+        <translation>음악</translation>
     </message>
     <message>
         <location filename="../src/screensaver/screensaver.cpp" line="2521"/>
         <source>Photo Albun</source>
-        <translation type="unfinished"></translation>
+        <translation>사진 앨범</translation>
     </message>
     <message>
         <source>Set as desktop wallpaper</source>
@@ -3475,7 +3475,7 @@ remaining</source>
     <message>
         <location filename="../src/screensaver/screensaver.cpp" line="180"/>
         <source>View</source>
-        <translation type="unfinished"></translation>
+        <translation>미리보기</translation>
     </message>
     <message>
         <location filename="../src/ukccplugins/screensaver/screensaver.cpp" line="57"/>
@@ -3552,12 +3552,12 @@ remaining</source>
     <message>
         <location filename="../src/ukccplugins/screensaver/screensaverui.cpp" line="119"/>
         <source>Weather</source>
-        <translation type="unfinished"></translation>
+        <translation>날씨</translation>
     </message>
     <message>
         <location filename="../src/ukccplugins/screensaver/screensaverui.cpp" line="132"/>
         <source>Music</source>
-        <translation type="unfinished"></translation>
+        <translation>음악</translation>
     </message>
     <message>
         <location filename="../src/ukccplugins/screensaver/screensaverui.cpp" line="145"/>
@@ -3784,7 +3784,7 @@ remaining</source>
     <message>
         <location filename="../src/screensaver/sleeptime.cpp" line="75"/>
         <source>You have rested</source>
-        <translation type="unfinished"></translation>
+        <translation>당신은 휴식을 취했습니다</translation>
     </message>
 </context>
 <context>
@@ -3884,7 +3884,7 @@ remaining</source>
     <message>
         <location filename="../src/widgets/talkswidget.cpp" line="58"/>
         <source>Enter your answer</source>
-        <translation type="unfinished"></translation>
+        <translation>답을 입력하세요</translation>
     </message>
 </context>
 <context>
@@ -3893,13 +3893,13 @@ remaining</source>
         <location filename="../src/widgets/userlistwidget.cpp" line="67"/>
         <location filename="../src/widgets/userlistwidget.cpp" line="121"/>
         <source>Login</source>
-        <translation type="unfinished"></translation>
+        <translation>로그인</translation>
     </message>
     <message>
         <location filename="../src/widgets/userlistwidget.cpp" line="69"/>
         <location filename="../src/widgets/userlistwidget.cpp" line="123"/>
         <source>Guest</source>
-        <translation type="unfinished"></translation>
+        <translation>관광객</translation>
     </message>
 </context>
 <context>
@@ -3921,28 +3921,28 @@ remaining</source>
     <message>
         <location filename="../src/widgets/verifykyidwidget.cpp" line="45"/>
         <source>Enter your Kylin ID account</source>
-        <translation type="unfinished"></translation>
+        <translation>키린 ID 계정 입력</translation>
     </message>
     <message>
         <location filename="../src/widgets/verifykyidwidget.cpp" line="64"/>
         <source>Input password</source>
-        <translation type="unfinished"></translation>
+        <translation>비밀번호를 입력하세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/verifykyidwidget.cpp" line="77"/>
         <source>Enter the verification code</source>
-        <translation type="unfinished"></translation>
+        <translation>인증 코드를 입력하세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/verifykyidwidget.cpp" line="84"/>
         <location filename="../src/widgets/verifykyidwidget.cpp" line="109"/>
         <source>Get verification code</source>
-        <translation type="unfinished"></translation>
+        <translation>인증 코드를 받으세요</translation>
     </message>
     <message>
         <location filename="../src/widgets/verifykyidwidget.cpp" line="112"/>
         <source>Reacquire(%1)</source>
-        <translation type="unfinished"></translation>
+        <translation>재획득 (%1)</translation>
     </message>
 </context>
 <context>
@@ -4036,7 +4036,7 @@ remaining</source>
     <message>
         <location filename="../src/lock-dialog/lockdialogmodel.cpp" line="69"/>
         <source>how long to show lock</source>
-        <translation type="unfinished"></translation>
+        <translation>잠금 화면이 표시되는 데 얼마나 걸리나요?</translation>
     </message>
 </context>
 <context>
@@ -4044,7 +4044,7 @@ remaining</source>
     <message>
         <location filename="../src/lock-dialog/lockdialogmodel.cpp" line="74"/>
         <source>if show lock</source>
-        <translation type="unfinished"></translation>
+        <translation>잠겨 있나요?</translation>
     </message>
 </context>
 <context>
@@ -4052,29 +4052,29 @@ remaining</source>
     <message>
         <location filename="../src/lock-command/main.cpp" line="44"/>
         <source>Start command for the ukui ScreenSaver.</source>
-        <translation type="unfinished"></translation>
+        <translation>명령어의 잠금 화면을 활성화하세요</translation>
     </message>
     <message>
         <location filename="../src/lock-command/main.cpp" line="49"/>
         <location filename="../src/lock-dialog/lockdialogmodel.cpp" line="53"/>
         <location filename="../src/lock-dialog/lockdialogmodel.cpp" line="55"/>
         <source>lock the screen immediately</source>
-        <translation type="unfinished"></translation>
+        <translation>즉시 화면을 잠가세요</translation>
     </message>
     <message>
         <location filename="../src/lock-command/main.cpp" line="52"/>
         <source>query the status of the screen saver</source>
-        <translation type="unfinished"></translation>
+        <translation>잠금 화면 상태 조회하기</translation>
     </message>
     <message>
         <location filename="../src/lock-command/main.cpp" line="55"/>
         <source>unlock the screen saver</source>
-        <translation type="unfinished"></translation>
+        <translation>화면 잠금 해제</translation>
     </message>
     <message>
         <location filename="../src/lock-command/main.cpp" line="57"/>
         <source>show the screensaver</source>
-        <translation type="unfinished"></translation>
+        <translation>디스플레이 스크린세이버</translation>
     </message>
     <message>
         <location filename="../src/lock-command/main.cpp" line="60"/>
@@ -4085,7 +4085,7 @@ remaining</source>
         <location filename="../src/lock-command/main.cpp" line="63"/>
         <location filename="../src/lock-dialog/lockdialogmodel.cpp" line="77"/>
         <source>show the session tools</source>
-        <translation type="unfinished"></translation>
+        <translation>디스플레이 세션 관리</translation>
     </message>
     <message>
         <location filename="../src/lock-command/main.cpp" line="65"/>
@@ -4096,13 +4096,13 @@ remaining</source>
         <location filename="../src/lock-command/main.cpp" line="68"/>
         <location filename="../src/lock-dialog/lockdialogmodel.cpp" line="87"/>
         <source>show the app block window</source>
-        <translation type="unfinished"></translation>
+        <translation>애플리케이션 차단 종료 인터페이스 표시</translation>
     </message>
     <message>
         <location filename="../src/lock-command/main.cpp" line="73"/>
         <location filename="../src/lock-dialog/lockdialogmodel.cpp" line="92"/>
         <source>show the multiUsers block window</source>
-        <translation type="unfinished"></translation>
+        <translation>다중 사용자 차단 종료 인터페이스 표시</translation>
     </message>
     <message>
         <location filename="../src/lock-backend/main.cpp" line="63"/>
